app/presenters/idv/in_person/ready_to_verify_presenter.rb

```diff
@@ -1,31 +1,19 @@
-require 'barby'
-require 'barby/barcode/code_128'
-require 'barby/outputter/png_outputter'
-
 module Idv
   module InPerson
     class ReadyToVerifyPresenter
       # WILLFIX: With LG-6881, confirm timezone or use deadline from enrollment response.
       USPS_SERVER_TIMEZONE = ActiveSupport::TimeZone['America/New_York']
 
-      delegate :selected_location_details, to: :enrollment
+      delegate :selected_location_details, :enrollment_code, :profile, to: :enrollment
 
       def initialize(enrollment:)
         @enrollment = enrollment
       end
 
-      def barcode_data_url
-        "data:image/png;base64,#{Base64.strict_encode64(barcode_image_data)}"
-      end
-
       def formatted_due_date
         due_date.in_time_zone(USPS_SERVER_TIMEZONE).strftime(I18n.t('time.formats.event_date'))
       end
 
-      def formatted_enrollment_code
-        EnrollmentCodeFormatter.format(enrollment_code)
-      end
-
       def selected_location_hours(prefix)
```

Comment:
Hm, if I make those changes here before #6624 is merged, then it would break the page rendering for "Ready to verify"? 🤔 

I guess I can plug the same changes noted previously into this code:

https://github.com/18F/identity-idp/blob/17d7e30e24c4cacb7801fe92e313c2c82e09338a/app/controllers/api/verify/password_confirm_controller.rb#L101-L120

Reply:
@tomas-nava Yeah, there's a lot from #6624 that I'd have to pull into this branch to be able to use the updated selected location shape. Could we merge this without those and update #6624? Or, conversely, merge #6624 and I'll update this one?

Reply:
I'm fine with merging this first and updating in #6624 @aduth